src/main/java/io/github/moulberry/notenoughupdates/util/Utils.java

```diff
@@ -1983,6 +1979,42 @@ public static void drawRectNoBlend(int left, int top, int right, int bottom, int
 		GlStateManager.enableTexture2D();
 	}
 
+	/**
+	 * Draws a solid color rectangle with the specified coordinates and color (ARGB format). Args: x1, y1, x2, y2, color
+	 * @see Gui#drawRect
+	 */
+	public static void drawRect(float left, float top, float right, float bottom, int color) {
+		float i;
+		if (left < right) {
+			i = left;
+			left = right;
+			right = i;
+		}
+		if (top < bottom) {
+			i = top;
+			top = bottom;
+			bottom = i;
+		}
+		float f = (float)(color >> 24 & 0xFF) / 255.0f;
+		float g = (float)(color >> 16 & 0xFF) / 255.0f;
+		float h = (float)(color >> 8 & 0xFF) / 255.0f;
+		float j = (float)(color & 0xFF) / 255.0f;
+		Tessellator tessellator = Tessellator.getInstance();
+		WorldRenderer worldRenderer = tessellator.getWorldRenderer();
+		GlStateManager.enableBlend();
+		GlStateManager.disableTexture2D();
+		GlStateManager.tryBlendFuncSeparate(770, 771, 1, 0);
+		GlStateManager.color(g, h, j, f);
+		worldRenderer.begin(7, DefaultVertexFormats.POSITION);
```

Comment:
Please do not use magic constants. (be aware that the constant names suggested below might not fully add up)
```suggestion
		GlStateManager.tryBlendFuncSeparate(GL_SRC_ALPHA, GL_ONE_MINUS_SRC_ALPHA, 1, 0);
		GlStateManager.color(g, h, j, f);
		worldRenderer.begin(GL_QUADS, DefaultVertexFormats.POSITION);
```

Reply:
resolved all of them in `Utils.java` :)